src/main/java/io/github/thebusybiscuit/sensibletoolbox/slimefun/SlimefunBridge.java

```diff
@@ -31,6 +31,8 @@
 
 public final class SlimefunBridge implements SlimefunAddon {
 
+    public static final RecipeType RECIPE_TYPE_MOB_DROP = new RecipeType(new NamespacedKey(SensibleToolboxPlugin.getInstance(), "mob_drop"), new CustomItem(Material.IRON_SWORD, "&bMob Drop", "&7Kill that Mob to", "&7obtain this Item"));
```

Comment:
This should not be static, move it to the other recipe types below please.

Reply:
Done, thanks